src/main/java/org/spongepowered/common/mixin/core/server/MixinMinecraftServer.java

```diff
@@ -531,6 +530,33 @@ protected void prepareSpawnArea(WorldServer world) {
     }
 
     @Override
+    public Optional<World> copyWorld(World originalWorld, String copyName) {
+        try {
+            FileUtils.copyDirectory(getWorldDir(originalWorld.getName()), getWorldDir(copyName));
```

Comment:
No because players could modify the world in-between, it wouldn't be saved,
I might add that as a separate copy but it shouldn't be the default kind,
because the most likely case I see is duplication of a minigame world to
clean up after use. And in that case you're going to want players out of
the world anyway.